src/mame/jpm/jpmsru.cpp

```diff
@@ -1270,6 +1279,8 @@ void jpmsru_state::lc(machine_config &config)
 {
 	jpmsru_6k(config);
 	m_maincpu->set_addrmap(AS_IO, &jpmsru_state::outputs_lc);
+
+	DAC_1BIT(config, "dac", 0).add_route(ALL_OUTPUTS, "mono", 0.5);
 }
```

Comment:
Likewise, this machine configuration member function should be moved to the class with the DAC, and you can use `m_dac` rather than repeating the literal tag here to avoid the potential for the tags to get out-of-sync.

Reply:
Fixed.